src/Analyzer.Core.BuiltInRuleTests/TestRunner.cs

```diff
@@ -54,6 +54,11 @@ public void TestRule(TestConfiguration ruleExpectations)
                                     .ToList())
                     .ToHashSet();
 
+                failingLines.UnionWith(thisRuleEvaluation.Results
+                .Where(r => !r.Passed)
+                .Select(r => r.LineNumber)
+                .ToHashSet());
+
```

Comment:
Should these be removed since they were in Yanelis' PR - merge conflicts?

Reply:
If I remove it the tests don't run properly, I'll handle the merge after Yanelis merges hers to main